src/database/postgres_database.rs

```diff
@@ -30,7 +28,8 @@ pub async fn connect() -> Result<PgPool, sqlx::Error> {
     Ok(pool)
 }
 pub async fn check_for_migrations() -> Result<(), sqlx::Error> {
-    let uri = &*dotenv::var("DATABASE_URL").expect("`DATABASE_URL` not in .env");
+    let uri = dotenv::var("DATABASE_URL").expect("`DATABASE_URL` not in .env");
```

Comment:
why not declare uri in one line rather than shadowing it?

Reply:
yeah didn't understand what i was doing there lol

Reply:
Oh yikes got a [`E0716`](https://doc.rust-lang.org/error-index.html#E0716) when I changed that to one line, so I guess I *need* to make that two lines. I have no idea why but that's how it is